#Define the goal

uint8 DRIVE_TASK   = 0
uint8 REVERSE_TASK = 1
uint8 task

uint8 POSE_TYPE = 1
uint8 PATH_TYPE = 2
uint8 FILE_TYPE = 3
uint8 type

geometry_msgs/Pose2D   target_pose
geometry_msgs/Pose2D[] target_path

float32 path_resolution
float32 expect_speed
string  roadnet_file
bool    path_filp

---
#Define the result
bool success

---

#Define a feedback message
float32 speed
float32 steer_angle
float32 percentage
